pkg/controller/rsm/transformer_object_generation.go

```diff
@@ -538,14 +538,30 @@ func injectRoleProbeBaseContainer(rsm workloads.ReplicatedStateMachine, template
 		return nil
 	}
 
+	tryToGetLorryGrpcPort := func(container *corev1.Container) int {
+		for i, port := range container.Ports {
+			if port.Name == constant.LorryGRPCPortName {
+				return int(container.Ports[i].ContainerPort)
```

Comment:
container.Ports[i].ContainerPort may be nil or negative ?

Reply:
Yep, this case may occur when there are multiple lorry containers.